functions/src/modules/pm/categories/prizes/listPrize.ts

```diff
@@ -0,0 +1,57 @@
+import type { ExpressFunction } from "../../../../@types";
+import { firebaseApp } from "../../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../../helpers";
+import { uasPermissionSwitch } from "../../../../systems/uas";
+
+const listPrize: ExpressFunction = (req, res, next) => {
```

Comment:
This should be plural.
```suggestion
const listPrizes: ExpressFunction = (req, res, next) => {
```